example/ios/RNMaplibreExample.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,14 +1,15 @@
 {
+  "originHash" : "e70d3525c8e2819a8b34f22909815dab5c700c25a06c32388f3930f7b3627768",
   "pins" : [
     {
       "identity" : "maplibre-gl-native-distribution",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/maplibre/maplibre-gl-native-distribution",
       "state" : {
-        "revision" : "ffda61e298c1490d4860d5184e80d618aaadc089",
-        "version" : "5.13.0"
+        "revision" : "6d0071977ed1f2380c739715f82ac650f99b0824",
+        "version" : "6.4.0"
       }
     }
   ],
-  "version" : 2
+  "version" : 3
```

Comment:
@sjchristi, Not sure why, but this change here causes xcode build errors for me on Xcode 15.2. Switching this back to version 2, seems to resolve the issue.

Reply:
Ok.  I'm on xcode 15.3, it probably made that switch.  I can change it back

Reply:
Are you building with Xcode or the command line tools?  We need to make sure the example builds with `react-native run-ios` and `react-native run-android`.

Reply:
I tested the sample app from command line.  For my person app, I tested from both command line and directly from xcode.